inference/core/workflows/core_steps/models/foundation/florence_2.py

```diff
@@ -0,0 +1,274 @@
+    WorkflowParameterSelector,
+)
+from inference.core.workflows.prototypes.block import (
+    BlockResult,
+    WorkflowBlock,
+    WorkflowBlockManifest,
+)
+
+TASKS_WITH_PROMPT = [
+    "<CAPTION_TO_PHRASE_GROUNDING>",
+    "<REFERRING_EXPRESSION_SEGMENTATION>",
+    "<REGION_TO_SEGMENTATION>",
+    "<OPEN_VOCABULARY_DETECTION>",
+    "<REGION_TO_CATEGORY>",
+    "<REGION_TO_DESCRIPTION>",
+]
+
+
+class BlockManifest(WorkflowBlockManifest):
+    model_config = ConfigDict(
+        json_schema_extra={
+            "name": "Florence-2 Model",
+            "short_description": "Run a multitask transformer model for a wide range of computer vision tasks.",
+            "long_description": "Florence-2 is a multitask transformer model that can be used for a wide range of computer vision tasks. It is based on the Vision Transformer architecture and has been trained on a large-scale dataset of images with a wide range of labels. The model is capable of performing tasks such as image classification, object detection, and image segmentation.",
+            "license": "MIT",
+            "block_type": "model",
+        }
+    )
+    type: Literal["Florence2Model", "Florence2"]
+    images: Union[WorkflowImageSelector, StepOutputImageSelector] = ImageInputField
+    model_id: Union[WorkflowParameterSelector(kind=[ROBOFLOW_MODEL_ID_KIND]), str] = (
+        Field(
+            title="Model",
+            default="florence-pretrains/1",
+            description="Roboflow model identifier",
+            examples=["florence-pretrains/1", "$inputs.model"],
+        )
+    )
+    vision_task: Union[
+        Literal[
+            "<OD>",
+            "<CAPTION_TO_PHRASE_GROUNDING>",
+            "<DENSE_REGION_CAPTION>",
+            "<REGION_PROPOSAL>",
+            "<OCR_WITH_REGION>",
+            "<REFERRING_EXPRESSION_SEGMENTATION>",
+            "<REGION_TO_SEGMENTATION>",
+            "<OPEN_VOCABULARY_DETECTION>",
+            "<REGION_TO_CATEGORY>",
+            "<REGION_TO_DESCRIPTION>",
+        ],
+        WorkflowParameterSelector(kind=[STRING_KIND]),
+    ] = Field(
+        description="The computer vision task to perform.",
+        default="<OPEN_VOCABULARY_DETECTION>",
+        examples=["<OPEN_VOCABULARY_DETECTION>"],
+    )
+    prompt: Union[WorkflowParameterSelector(kind=[LIST_OF_VALUES_KIND]), List[str]] = (
+        Field(
+            description="The accompanying prompt for the task (comma separated).",
+            examples=[["red apple", "blue soda can"], "$inputs.prompt"],
+        )
+    )
+
+    @classmethod
+    def accepts_batch_input(cls) -> bool:
+        return True
+
+    @classmethod
+    def describe_outputs(cls) -> List[OutputDefinition]:
+        return [
+            OutputDefinition(name="parent_id", kind=[BATCH_OF_PARENT_ID_KIND]),
+            OutputDefinition(name="root_parent_id", kind=[BATCH_OF_PARENT_ID_KIND]),
+            OutputDefinition(name="image", kind=[BATCH_OF_IMAGE_METADATA_KIND]),
```

Comment:
this output is no longer used given sv.Detections are there under `predictions`

Reply:
raw output is the raw text from the model, structured output is the formatted dict generated by the microsoft provided output text processor, predictions is sv detections processed by our from_lmm method in supervision